src/bootstrap/config.rs

```diff
@@ -50,7 +50,7 @@ pub enum DryRun {
     UserSelected,
 }
 
-#[derive(Copy, Clone, Default)]
+#[derive(Copy, Clone, Default, Debug, PartialEq, Eq)]
```

Comment:
i am not sure how much these derives actually buy us. i don't think comparing the whole Config is particularly useful (see below for how the test is broken today) and it means that the compile times go up.